pkgs/build-support/deterministic-uname/deterministic-uname.sh

```diff
@@ -55,7 +55,7 @@ else
 fi
 
 # With no OPTION, same as -s.
-if [[ $# -eq 0 ]]; then
+if [[ "$1" = "--" ]]; then
```

Comment:
Not sure if this is the idiomatic way to check for lack of arguments with `getopts`. Another way to fix this would be to move this above `getopts`.

Reply:
moved it above getopts